### PR TITLE
get the float-away and sink-away icon mixins working

### PR DESCRIPTION
**Pull request by hippogriffic**
bit messy, they require that you have 2 identical svgs in the html

blah

blah

@incuna/frontend please review

### CODE REVIEW

**Comment by pandalion** [comment]
Cool! :D

File: hover/effects/icon-transitions/_icon-float-away.sass [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// This mixin requires that 2 identical icons to be present in the html
```
Comment (henrahmagix) [inline hover/effects/icon-transitions/_icon-float-away.sass:1]: `requires that 2 indentical icons to be present`:

Either
`requires that 2 indentical icons be present`
or
`requires 2 indentical icons to be present`

**Comment by henrahmagix** [comment]
👍

**Comment by grahamgilchrist** [comment]
ooh thats so pretty

**Pull request merged**